client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -183,4 +108,30 @@
 			}
 		}
 	}
+
+	.checkout-thank-you__gridicon-wrapper {
```

Comment:
@scruffian We don't need to include this selector inside the `.checkout-thank-you__purchase-detail` ruleset.